bundles/org.openhab.binding.jellyfin/NOTICE

```diff
@@ -0,0 +1,13 @@
+This content is produced and maintained by the openHAB project.
+
+* Project home: https://www.openhab.org
+
+== Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License 2.0 which is available at
+https://www.eclipse.org/legal/epl-2.0/.
+
+== Source Code
```

Comment:
The licenses of the used libraries need to be mentioned here.

Reply:
Fixed